changelog/6466.improvement.rst

```diff
@@ -0,0 +1,6 @@
+[Response selector](retrieval-actions.mdx) is now trained on retrieval intent labels by default instead of actual response text.
+
+If you want to use the actual response text for training, then use ``train_on_text=True`` in the configuration of ``ResponseSelector``.
```

Comment:
Can we maintain the description of the features as described here in the changelog, but also mention in the migration guide _why_ this change was made (`For most models, this should improve training time of the component with similar confidence results.`, if that's accurate) as well as how to revert to the default? It'd be good to have this info whenever we change defaults that we know could actually be bad for peoples' models, e.g. tom's example of chitchat/faq_1, chitchat/faq_2. example below:

Reply:
[Response Selectors](retrieval-actions.mdx) are now trained on retrieval intent labels by default instead of actual response text. For most models, this should improve training time of the `ResponseSelector`, with similar confidence results.

If you want to revert to the pre-2.0 default behavior, add the `train_on_text: true` parameter to your `ResponseSelector` component.